### PR TITLE
Source Salesforce: include primary key in every chunk

### PR DESCRIPTION
**Pull request by davydov-d**
## What
https://github.com/airbytehq/airbyte/issues/20703
*this is a followup for the on-call issue https://github.com/airbytehq/oncall/issues/1403*

## How
If the properties are chunked - we need to include the primary key in every chunk, otherwise we won't be able to stick response chunks

### CODE REVIEW

File: airbyte-integrations/connectors/source-salesforce/source_salesforce/streams.py [hunk 1]
```diff
@@ -136,13 +136,16 @@ def request_params(
     def chunk_properties(self) -> Iterable[Mapping[str, Any]]:
         selected_properties = self.get_json_schema().get("properties", {})
 
+        def empty_props():
```
Comment (bazarnov) [inline airbyte-integrations/connectors/source-salesforce/source_salesforce/streams.py:139]: This should have another naming, how about `include_primary_key()`?

**Review by bazarnov — APPROVED** [review]

**Review by maxi297 — APPROVED** [review]
LGTM!

**Review by bazarnov — APPROVED** [review]

File: airbyte-integrations/connectors/source-salesforce/source_salesforce/streams.py (hunk 1 above)
Comment (davydov-d) [reply airbyte-integrations/connectors/source-salesforce/source_salesforce/streams.py]: renamed

**Review by lazebnyi — APPROVED** [review]

**Pull request merged**